rpcs3/Emu/Cell/lv2/sys_spu.cpp

```diff
@@ -405,11 +405,11 @@ error_code sys_spu_thread_initialize(ppu_thread& ppu, vm::ptr<u32> thread, u32 g
 
 	verify(HERE), idm::import<named_thread<spu_thread>>([&]()
 	{
-		std::string full_name = fmt::format("SPU[0x%07x] Thread", tid);
+		std::string full_name = fmt::format("SPU[0x%07x] ", tid);
 
 		if (!thread_name.empty())
 		{
-			fmt::append(full_name, " (%s)", thread_name);
+			fmt::append(full_name, "%s ", thread_name);
```

Comment:
```suggestion
			fmt::append(full_name, " %s", thread_name);
```

Reply:
I actually want that space infront as I found it odd looking if the last character is a normal character in the rectangle with no blank space inbetween.
It also applies for threads with "empty" names.
I wonder if there's a qt way of doing it instead.

Reply:
do you have a screenshot of the "weird" names